docs/tutorials/defining-hamiltonians.md

```diff
@@ -1,36 +1,28 @@
 # Defining Hamiltonians
 
-In this short tutorial, we explain how to define Hamiltonians in dynamiqs. There are currently three ways: using array-like objects for constant Hamiltonians, defining a function for time-dependent Hamiltonians, and using a custom list format for piecewise constant Hamiltonians.
+In this short tutorial, we explain how to define Hamiltonians in dynamiqs. There are currently four ways: using array-like objects for constant Hamiltonians, defining a function or function-like coefficients for time-dependent Hamiltonians, and using a custom format for piecewise constant Hamiltonians.
```

Comment:
"defining a function or function-like coefficients for time-dependent Hamiltonians" > unclear, I think you can split both cases with a coma and explain individually.